### DISCUSSION TITLE
telegram not valid

### DISCUSSION THREAD

**Discussion by Horiodino** [deckhouse/deckhouse#5231] [Q&A] [answered]
hey guys i am Praful , i found the project very interesting so i was looking to contribute ,
im trying to join the telegram but unable to join can you please provide the link again.

**Comment by shurup** [comment, marked as answer]
Hi @Horiodino! Here's the correct link: https://t.me/deckhouse — do you have any issues with it?

**Reply by Horiodino** [reply]
Thank you for providing the link but I'm still not able to join the channel 
If it's possible can you add me please here's my telegram username --   HoliOdin
